mmhuman3d/data/data_structures/human_data.py

```diff
@@ -24,6 +24,9 @@
     'image_path': {
         'type': list,
     },
+    'image_id': {
```

Comment:
@LazyBusyYang Please help review this new added key

Reply:
Looks good. As long as it's similar to `image_path`.